pkg/reconciler/dependencybuild/scripts/gradle-build.sh

```diff
@@ -69,7 +66,8 @@ echo "Running Gradle command with arguments: $@"
 if [ ! -d $(workspaces.source.path)/source ]; then
   cp -r $(workspaces.source.path)/workspace $(workspaces.source.path)/source
 fi
-gradle -Dmaven.repo.local=$(workspaces.source.path)/artifacts --info --stacktrace "$@"  | tee $(workspaces.source.path)/logs/gradle.log
+# release.version is only for Nebula Release plugin.
+gradle -Dmaven.repo.local=$(workspaces.source.path)/artifacts --info --stacktrace -Prelease.version=$(params.PROJECT_VERSION) "$@" | tee $(workspaces.source.path)/logs/gradle.log
```

Comment:
Can this go into build discovery instead so it ends up in the recipe? Also in general we should be referencing the env var version of PROJECT_VERSION rather than using the params. We should try and remove all copies of $(params) from the scripts and replace them with env vars at some point.